scripts/helper/chron-ocs.sh

```diff
@@ -21,59 +21,9 @@ else
     echo "$CHRONOCS stopped"
 fi
 
-# Before launching the steps we need to check prerequisites and credentials:
```

Comment:
This is showing deleted code that is not included in the project.   When I integrated the code from Carlos, I rewrote it eliminating this code, so you must be working from an old copy.   Anyway, I will try to integrate it as it says it will not be included.